grouper/models/base/session.py

```diff
@@ -31,7 +34,12 @@ def wrapper(self, *args, **kwargs):
 
 def get_db_engine(url):
     # type: (str) -> Engine
-    return create_engine(url, pool_recycle=300)
+    try:
+        engine = create_engine(url, pool_recycle=300)
+    except (ArgumentError, OperationalError):
+        logging.exception("Can't create database engine.")
```

Comment:
The lack of detail here would make debugging such an exception quite difficult. I'm not thinking of a great way to safely log anything though, in case of arbitrarily malformed values of `url` (that still contain the URL)...

Reply:
Yeah, I played around various ideas for a bit but I couldn't think of a solution to safely catch and sanitize all outputs.